File: Day 1/fizzbuzz.py
def fizz_buzz(i):
  if i%15 == 0: #Test for FizzBuzz
    return("FizzBuzz")    
  elif i%3 == 0: #Test for Fizz
    return("Fizz")
  elif i%5 == 0: #Test for Buzz
    return("Buzz")
  else:  #Condition if not Fizz,Buzz or FizzBuzz
    return (i)

File: Other Andelabs/fizzbuzz.js
function fizzBuzz(i)
{
    //test for fizzbuzz
    if (i%15 === 0){
        return ("FizzBuzz")    
    }
    //test for fizz
    else if ( i%3 === 0){
        return ("Fizz")
    }
    //test for buzz
    else if (i%5 === 0) {
        return ("Buzz")
        }
    //default statement    
    else {
        return (i);
    }
}
